module/VuFindSearch/src/VuFindSearch/Backend/EDS/Base.php

```diff
@@ -447,11 +478,22 @@ protected function call(
             'Content-Type' => $this->contentType,
             'Accept-Encoding' => 'gzip,deflate',
         ];
+        $this->debug(
+            'isguest: ' . $this->isGuest
+            . ' | APIKey: ' . $this->apiKey
+            . ' | APIKey Guest: ' . $this->apiKeyGuest
```

Comment:
Since some of these values might be non-strings, I'd suggest:

```suggestion
            'isguest: ' . ($this->isGuest ? 'true' : 'false')
            . ' | APIKey: ' . ($this->apiKey ?? '-')
            . ' | APIKey Guest: ' . ($this->apiKeyGuest ?? '-')
```

Reply:
But it might also be worth using `substr` to truncate the keys when set if you don't want to leak out the full things (though without knowing what a real one looks like, I'm not sure exactly what to suggest).